elasticdl_preprocessing/layers/concatenate_with_offset.py

```diff
@@ -0,0 +1,66 @@
+import tensorflow as tf
+
+
+class ConcatenateWithOffset(tf.keras.layers.Concatenate):
+    """Layer that add offset for tensor in the list of inputs and
```

Comment:
Layer that add offset to each id tensor in the input list and then concatenate these tensors.